src/components/DayPicker.jsx

```diff
@@ -516,7 +535,7 @@ export default class DayPicker extends React.Component {
     this.setState({
       currentMonth: newMonth,
       monthTransition: null,
-      translationValue: 0,
+      translationValue: (this.props.isRTL && this.isHorizontal()) ? translationValueRTL : 0,
```

Comment:
I don't know that this should be non-zero here even if it is RTL since this is after the transition.